core/services/fluxmonitor/flux_monitor.go

```diff
@@ -736,17 +740,19 @@ func (p *PollingDeviationChecker) processLogs() {
 			continue
 		}
 
+		ctx, cancel = postgres.DefaultQueryCtx()
+		defer cancel()
```

Comment:
Perhaps a bit cleaner if we do `db := p.store.DB.WithContext(ctx)` here?

Reply:
not sure I follow - just to give it a name?